src/app/src/components/FacilityDetailSidebar.jsx

```diff
@@ -194,24 +217,53 @@ class FacilityDetailSidebar extends Component {
                             label="Contributors:"
                             isContributorsList
                         />
+                        <FeatureFlag flag={CLAIM_A_FACILITY}>
+                            <ShowOnly when={!!data.properties.claim_info}>
+                                <FacilityDetailSidebarClaimedInfo
+                                    data={data.properties.claim_info}
+                                />
+                            </ShowOnly>
+                        </FeatureFlag>
                         <div className="control-panel__group">
                             <div style={detailsSidebarStyles.linkSectionStyle}>
                                 <a
```

Comment:
The wireframes show conditionally hiding these links if the logged in contributor has claimed the facility, since the do not apply to the claimant. Instead, the wireframes have a link to the profile editing page above the satellite image. Since we already have access to `facilityIsClaimedByCurrentUser` we should consider implementing this link swapping.

Reply:
Handled in be9749dfe4cbb9fbf4a31df0a72425441763cee5

Not my claimed facility:

<img width="463" alt="Screen Shot 2019-06-13 at 11 34 49 AM" src="https://user-images.githubusercontent.com/4165523/59446445-8162e700-8dcf-11e9-90bc-43e08d3da29b.png">

My claimed facility:

![Screen Shot 2019-06-13 at 11 34 54 AM](https://user-images.githubusercontent.com/4165523/59446459-8889f500-8dcf-11e9-9607-a664ad70459c.png)

Reply:
Note that the verified badge will already also conditionally link to the facility claim form if it is my claimed facility.